Tests/MaterialLib/TestRelativePermeabilityModel.cpp

```diff
@@ -0,0 +1,201 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * \file   TestRelativePermeabilityModel.cpp
+ *
+ * Created on November 2, 2016, 3:09 PM
+ */
+
+#include <gtest/gtest.h>
+
+#include <memory>
+#include <vector>
+
+#include "BaseLib/ConfigTree.h"
+
+#include "TestTools.h"
+
+#include "MaterialLib/PorousMedium/UnsaturatedProperty/RelativePermeability/RelativePermeability.h"
+#include "MaterialLib/PorousMedium/UnsaturatedProperty/RelativePermeability/CreateRelativePermeabilityModel.h"
+
+using namespace MaterialLib;
+using namespace MaterialLib::PorousMedium;
+
+std::unique_ptr<RelativePermeability> createRelativePermeabilityModel(
+    const char xml[])
+{
+    auto const ptree = readXml(xml);
+    BaseLib::ConfigTree conf(ptree, "", BaseLib::ConfigTree::onerror,
+                             BaseLib::ConfigTree::onwarning);
+    auto const& sub_config = conf.getConfigSubtree("relative_permeability");
+    sub_config.ignoreConfigAttribute("id");
+    return MaterialLib::PorousMedium::createRelativePermeabilityModel(
```

Comment:
If you are 'using namespace MaterialLib::PorousMedium' than this can be shorter. Choose one variant.